### PR TITLE
Add ResponsivelyApp

### PR DESCRIPTION
**Pull request by QAInsights**
Add ResponsivelyApp

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1423)

### CODE REVIEW

File: manifests/ResponsivelyApp/ResponsivelyApp/0.1.6.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Name: ResponsivelyApp
+Publisher: ResponsivelyApp
+License: © 2020 ResponsivelyApp, Inc. All rights reserved.
+AppMoniker: ResponsivelyApp
+Tags: responsively,app,viewpoint
+Description: A modified browser built using Electron that helps in responsive web development.
+Homepage: https://manojvivek.github.io/responsively-app/
+Installers:
+  - Arch: x64
+    Url: https://github.com/manojVivek/responsively-app/releases/download/v0.1.6/ResponsivelyApp-Setup-0.1.6.exe
```
Comment (MarNicGit) [inline manifests/ResponsivelyApp/ResponsivelyApp/0.1.6.yaml:12]: This URL 404's, according to the [releases](https://github.com/manojVivek/responsively-app/releases) page of ResponsivelyApp, version 0.1.6 hasn't been released yet.

Did you mean to add version 0.1.5? (https://github.com/manojVivek/responsively-app/releases/download/v0.1.5/ResponsivelyApp-Setup-0.1.5.exe)

Comment (QAInsights) [reply manifests/ResponsivelyApp/ResponsivelyApp/0.1.6.yaml]: Actually it was 0.1.6, now the author changed to 0.1.5. I have updated the correct version.

File: manifests/ResponsivelyApp/ResponsivelyApp/0.1.5.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Publisher: ResponsivelyApp
+License: © 2020 ResponsivelyApp, Inc. All rights reserved.
+AppMoniker: ResponsivelyApp
+Tags: responsively,app,viewpoint
+Description: A modified browser built using Electron that helps in responsive web development.
+Homepage: https://manojvivek.github.io/responsively-app/
+Installers:
+  - Arch: x64
+    Url: https://github.com/manojVivek/responsively-app/releases/download/v0.1.5/ResponsivelyApp-Setup-0.1.5.exe
+    Sha256: b359f668eee2f28847b9fc5acc211877509adaa0f6e663b24f0a659b1c2a6686
```
Comment (MarNicGit) [inline manifests/ResponsivelyApp/ResponsivelyApp/0.1.5.yaml:13]:
```suggestion
    Sha256: F27E6E72E2A921AF897DF6646153FB3AF0808D7348BFF96F97E0D94A1FDEC88C
```

This should be the correct hash for the 0.1.5 installer, according to `Get-FileHash`.

Comment (QAInsights) [reply manifests/ResponsivelyApp/ResponsivelyApp/0.1.5.yaml]: Yes, I just changed. Thanks for your inputs :)

**Comment by MarNicGit** [comment]
@QAInsights Just tried to install the manifest and ran into SmartScreen - You'll want to submit the installer to https://www.microsoft.com/en-us/wdsi/filesubmission and then post the results here once it's processed.

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**